Sources/ArcGISToolkit/Components/OfflineMapAreasView/OfflineMapAreasView.MapViewModel.swift

```diff
@@ -49,31 +42,23 @@ extension OfflineMapAreasView {
             
             preplannedMapModels = await Result {
                 try await offlineMapTask.preplannedMapAreas
+                    .filter { $0.id != nil }
                     .sorted(using: KeyPathComparator(\.portalItem.title))
-                    .compactMap {
+                    .map {
                         PreplannedMapModel(
                             offlineMapTask: offlineMapTask,
                             mapArea: $0,
-                            portalItemID: portalItemID
+                            portalItemID: portalItemID,
+                            preplannedMapAreaID: $0.id!.rawValue
                         )
                     }
             }
-            if let models = try? preplannedMapModels!.get() {
-                hasPreplannedMapAreas = !models.isEmpty
-                
-                for model in models {
-                    model.setMobileMapPackage()
-                }
-            }
         }
         
         /// Request authorization to show notifications.
-        func checkCanShowNotifications() async {
-            canShowNotifications = (
-                try? await UNUserNotificationCenter.current()
-                    .requestAuthorization(options: [.alert, .sound])
-            )
-            ?? false
+        func requestUserNotificationAuthorization() async {
```

Comment:
the name of this function did not match the description, nor did it match what it did.